packages-content-model/roosterjs-content-model-core/lib/publicApi/table/normalizeTable.ts

```diff
@@ -75,6 +79,8 @@ export function normalizeTable(
     for (let i = 0; i < columns; i++) {
         if (table.widths[i] === undefined) {
             table.widths[i] = getTableCellWidth(columns);
+        } else if (table.widths[i] < MIN_WIDTH) {
```

Comment:
Then here you can just do
else {
 table.width[i] = getEffectiveTableCellWidth(table.width[i]);
}